<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZFlyEmRoiToolDialog</class>
 <widget class="QDialog" name="ZFlyEmRoiToolDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ROI Tool</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="menuPushButton">
     <property name="text">
      <string>Menu</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Project</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="projectComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>---</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="newPushButton">
       <property name="text">
        <string>New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clonePushButton">
       <property name="text">
        <string>Clone</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Step</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="stepSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="prevPushButton">
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextPushButton">
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="show3dPushButton">
       <property name="text">
        <string>Show 3D</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nearestPushButton">
       <property name="text">
        <string>Locate Nearest</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="estimatePushButton">
       <property name="text">
        <string>Estimate ROI</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="uploadPushButton">
       <property name="text">
        <string>Upload</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dvidRoiPushButton">
       <property name="text">
        <string>Create DVID ROI</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_28">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Scale</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_29">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QPushButton" name="xyDecButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../gui.qrc">
             <normaloff>:/images/squeeze.png</normaloff>:/images/squeeze.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="xyIncButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../gui.qrc">
             <normaloff>:/images/stretch.png</normaloff>:/images/stretch.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_30">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QPushButton" name="xDecButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../gui.qrc">
             <normaloff>:/images/hsqueeze.png</normaloff>:/images/hsqueeze.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="xIncButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../gui.qrc">
             <normaloff>:/images/hstretch.png</normaloff>:/images/hstretch.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_31">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QPushButton" name="yDecButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../gui.qrc">
             <normaloff>:/images/vsqueeze.png</normaloff>:/images/vsqueeze.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="yIncButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../gui.qrc">
             <normaloff>:/images/vstretch.png</normaloff>:/images/vstretch.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="scaleStepDoubleSpinBox">
           <property name="toolTip">
            <string>Scale Step</string>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>0.001000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>0.050000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_32">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Move</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_33">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QPushButton" name="movexyDecButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../gui.qrc">
             <normaloff>:/images/move_upleft.png</normaloff>:/images/move_upleft.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="movexyIncButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../gui.qrc">
             <normaloff>:/images/move_downright.png</normaloff>:/images/move_downright.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_34">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QPushButton" name="movexDecButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../gui.qrc">
             <normaloff>:/images/go-previous.png</normaloff>:/images/go-previous.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="movexIncButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../gui.qrc">
             <normaloff>:/images/go-next.png</normaloff>:/images/go-next.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_35">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QPushButton" name="moveyDecButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../gui.qrc">
             <normaloff>:/images/move_up.png</normaloff>:/images/move_up.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="moveyIncButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../gui.qrc">
             <normaloff>:/images/move_down.png</normaloff>:/images/move_down.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="moveStepSpinBox">
           <property name="toolTip">
            <string>Move Step</string>
           </property>
           <property name="statusTip">
            <string/>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_36">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_37">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Rotate</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_38">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QPushButton" name="rotateLeftButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../gui.qrc">
               <normaloff>:/images/rotate_left.png</normaloff>:/images/rotate_left.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="rotateRightButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../gui.qrc">
               <normaloff>:/images/rotate_right.png</normaloff>:/images/rotate_right.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="rotateStepDoubleSpinBox">
             <property name="toolTip">
              <string>Rotate Step</string>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>0.002000000000000</double>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.002000000000000</double>
             </property>
             <property name="value">
              <double>0.020000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_39">
         <item>
          <widget class="QPushButton" name="estimateVolumeButton">
           <property name="text">
            <string>Estimate Volume</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ZFlyEmMessageWidget" name="messageWidget" native="true"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZFlyEmMessageWidget</class>
   <extends>QWidget</extends>
   <header>flyem/zflyemmessagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ZFlyEmRoiToolDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ZFlyEmRoiToolDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
